src/index.html

```diff
@@ -4,13 +4,43 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post" class="form"
+    data-qa="big">
+
+      <label>
+        <img src="images/search.svg"
+         alt="search" class="img">
+        <input
+         class="input input-big"
+         type="search"
+         placeholder="Try “Los Angeles“"
```

Comment:
Use `&quot;` instead of " in the placeholder

Reply:
done